draft-ietf-scitt-architecture.md

```diff
@@ -567,10 +562,62 @@ Protected_Header = {
 
 Unprotected_Header = {
   ; TBD, Labels are temporary
-  ? 394 => [+ SCITT_Receipt]
+  ? 394 => [+ Receipt]
 }
 ~~~~
 
+## Receipts
+
+Receipts are based on COSE Signed Merke Tree Proofs ({{-COSEMTP}}) with an additional wrapper structure that adds the following information:
+
+- version: Receipt version number; this should be set to `0` for implementation of this document. We envision that future version of SCITT may add support for more complex receipts; for instance, registrations on multiple TS, receipts for dependency graphs and endorsements of Signed Claims, etc.
+- ts_identifier: The DID of the Transparency Service that issued the claim. Verifiers MAY use this DID as a key discovery mechanism to verify the COSE Merkle Root signature; in this case the verification is the same as for Signed Claims and the signer should include the Key ID header. Verifiers MUST support the `did:web` method, all other methods are optional.
+
+We also introduce the following requirements for the COSE signature of the Merkle Root:
+
+- The SCITT version header MUST be included and its value match the `version` field of the Receipt stucture.
+- The DID of issuer header (like in Signed Claims) MUST be included and its value match the `ts_identifier` field of the Receipt structure.
+- TS MAY include the Registration policy info header to indicate to verifiers what policies have been applied at the registration of this claim.
+- Since {{-COSEMTP}} uses optional headers, the `crit` header (id: 2) MUST be included and all SCITT-specific headers (version, issuer DID and Registration Policy) MUST be marked critical.
+
+The following registration policies are built-in and MAY be used by verifiers to help decide the trustworthiness of the Transparent Statement:
+
+- Registration time: the timestamp at which the TS has added this Signed Claim to its Registry
+- DID Manifest: the manifest that was retured by resolving the DID of the issuer at registration, according to the TS.
+- Sequence number: the sequence number of this statement with relation to other statements with the same issuer and feed. There is no guarantee that all Signed Statements are registered with contiguous sequence numbers; only that it is monotonic and follows the issuer sequence numbers.
+
+~~~ cddl
+Receipt = [
+    version: int,
+    ts_identifier: tstr,
+    proof: SignedMerkleTreeProof
+]
+
+; Additional protected headers in the COSE signed_tree_root of the SignedMerkleTreeProof
+Protected_Header = {
+  390 => int               ; SCITT Version
+  391 => tstr              ; DID of Issuer (required)
+  393 => RegistrationInfo  ; Registration policy information (optional)
+
+  ; Other COSE Signed Merkle Tree headers
+  ; (e.g. tree algorithm, tree size)
+
+  ; Optional standard COSE headers
+  2 => [+label]            ; Critical headers
```

Comment:
any header parameters/attributes that are mandatory? we can make them mandatory by being in the label 2 array.